src/core_modules/capture-core/components/LoadingMasks/LoadingMaskForPage.component.js

```diff
@@ -22,9 +22,7 @@ const LoadingMaskForPagePlain = (props: Props) => (
     <div
         className={props.classes.container}
     >
-        <LoadingMask
-            size={50}
-        />
+        <CircularLoader />
```

Comment:
NIT: Since it's no longer `LoadingMask` do you think we should rename to `CircularLoaderForPage`

Reply:
Same explanation as the one in the `LoadingMaskForButton.component.js` file